docs/3_setup_guide/Delft_Fiat/hydromt_fiat_gui.qmd

```diff
@@ -0,0 +1,229 @@
+      b) **Draw box**: Click on the map and draw a box. Double click will end the drawing process.
+      c) **SFINCS model domain**: If you aleady created a SFINCS model you can select the **/gis-folder**. Do <u>not</u> select the region file itself. It will be recognized automatically by the model.
+      d) **Upload file**: You can upload your own boundary in form of a vector file (e.g. *.gpkg*, *.geojson*) with a valid polygon shape. 
+
+  2. As soon you are done click "**Generate boundary**" and your model boundary is established.
+
+For further details please refer to @fig-model_boundaries.
+
+::: {#fig-model_boundaries}
+
+{{< video ../../_static/videos/hydromt_fiat/Generate_Boundaries.mp4 >}}
+
+Create model boundaries using the "Draw Polygon" method.
+
+:::
+
+## Quickbuild
+If you set up a model in the United States and do not want to use your own data you can simply choose to work with the **quickbuild**. 
+
+The **quickbuild** downloads the data from different sources:  
+
+* **Exposure**: National Structure Inventory (NSI)
+* **Vulnerability curves**: FEMA’s Hazus Program
+* **Social vulnerability index**/**equity**: Census api
+* **Roads**: Open Street Map (OSM)
+
+*GUI instructions*
+
+  1. Go to the **Model Boundary | Quickbuild** -tab and select any of the options in the drop-down menu to create the boundary (@fig-quickbuild).
+  2. After you generated the model boundaries you can select the quickbuild option.
+
+![Workflow to use the "Quickbuild"-option.](../../_static/figures/Quickbuild.svg){#fig-quickbuild}
+
+For further details please refer to @fig-nsi_quickbuild.
+
+::: {#fig-nsi_quickbuild}
+
+{{< video ../../_static/videos/hydromt_fiat/Quickbuild.mp4 >}}
+
+Quickbuild option in the United States using NSI exposure, FEMA Hazus Damage Curves and OSM roads.
+
+:::
+
+## Model from scratch: A simple model
+This section explains the creation of a simple model without additional input by the user. This can be set up globally or in the United States, after you created your model boundaries (please refer to the section *Create Model Boundary*).
+
+### Create the exposure data
+There are two different database sources to download the **exposure data** from, determined by the location of your model:
+
+1. **United States: National Structure Inventory (NSI)**
+2. **Global: Open Street Map (OSM)**
+
+Additionally, you can download **road data** from Open Street Map. 
+
+*GUI instructions*  
+To set up the exposure open the **"Exposure"**-tab on the top panel (@fig-exposure). 
+
+  1. Select the **"Asset Location"**-tab at the bottom panel.
+  2. For a U.S. based model start with **NSI** data and for a global model start with **OSM** data.
+  3. ***Only OSM***: Define a <u>damage currency (JRC damage curves default is Euro)</u> and the <u>finished floor height (meters)</u> for all assets (can be updated later on the individual asset level in the Exposure >> Finished Floor Height-tab) .
```

Comment:
I would remove "JRC damage curves" - just say (default is Euro), because the reader doesn't know what the JRC curves are yet.

Reply:
But wouldn't the user be interested about the data source of the curves? I can cover that also in the data input section.